app/views/layouts/_policy_sim.html.haml

```diff
@@ -1,5 +1,8 @@
 - url = url_for_only_path(:action => 'policy_sim_add')
 #main_div
+  - if @right_cell_text && !@explorer
```

Comment:
Adding a new if into the view, i.e. extra complexity

Reply:
I never add such conditions anywhere if I really don't have to.